libkbfs/test_stallers.go

```diff
@@ -106,9 +267,10 @@ func maybeStall(ctx context.Context, opName stallableOp,
 		return
 	}
 
-	v, ok := ctx.Value(stallKey).(bool)
-	if !ok || !v {
-		return
+	if stallKey != "" {
```

Comment:
might be clearer to make the empty string into a constant like `stallEverythingKey` or something.